### ISSUE TITLE
Error: <svg> attribute viewBox: Expected number, "0 0..

### ISSUE THREAD

**Issue by wrabit** [chenqingspring/vue-lottie#20]
Setting up as per the example and getting:

Error: <svg> attribute viewBox: Expected number, "0 0 undefined undefi…

Any ideas?

**Comment by Sweetyy** [comment]
Same problem .. i did exactly the same as the example and always have this error...

**Comment by patrick-sung** [comment]
should be 
`animationData: animationData.default`

**Comment by Sweetyy** [comment]
> should be
> `animationData: animationData.default`

Thanks a lot ! It works for me now :)

**Comment by sebbean** [comment]
damn me to! frigg! thanks github!

**Comment by Xiaoha0226** [comment]
It works for me too.

**Comment by Xiaoha0226** [comment]
It works for me too. Thanks!  I want to know the reason is  'lottie-web' version upgrade？

**Comment by seemcat** [comment]
@patrick-sung you saved me from something that's been blocking me the **entire** day. THANK YOU! 😭

**Comment by ZalgirisKaunas** [comment]
OMG. WHY THIS ISN'T IN THE DOCUMENTATION.

I literally wasted half of the day..

**Comment by lansolo99** [comment]
Yeah, had the same problem, please update the doc, I would never have figured out by myself if @patrick-sung didn't reveal the trick

**Comment by TimRChen** [comment]
How can I find this in the Lottie doc?

**Comment by alexsandro-xpt** [comment]
Ow please update the documentation( https://github.com/chenqingspring/vue-lottie/blob/master/README.md#L65 ) for `animationData.default` please!

**Comment by dhruvbhatia7** [comment]
Thanks @patrick-sung, was stuck on this for over an hour!

**Comment by jeancabral** [comment]
Thx so much

> should be
> `animationData: animationData.default`

Very thx! Save my day!

**Comment by carlossrb** [comment]
> should be
> `animationData: animationData.default`

thanks a _lottie_!!!

**Comment by cmdparkour** [comment]
> should be
> `animationData: animationData.default`

thanks for your help,

**Comment by Xiaoshizaizi** [comment]
> should be
> `animationData: animationData.default`

so cool

**Comment by DevidJoshua** [comment]
After

> should be
> `animationData: animationData.default`

Finally, thanks ya,,,  anyone tell me why it's not working as the documentation...

**Comment by pitometsyurii** [comment]
> After
> 
> > should be
> > `animationData: animationData.default`
> 
> Finally, thanks ya,,, anyone tell me why it's not working as the documentation...

Good question. But seems like it will not be fixed.

**Comment by yehee** [comment]
Could also change how it's imported:
```diff
- import * as animationData from './planets-spinning.json'
+ import animationData from './planets-spinning.json'
```
This way you don't have to get `animationData.default` as it automatically imports the default.

**Comment by cazroam** [comment]
2 years later and the documentation still needs updating :( 

There are even PRs for it: https://github.com/chenqingspring/vue-lottie/pull/40 https://github.com/chenqingspring/vue-lottie/pull/42 https://github.com/chenqingspring/vue-lottie/pull/51

@chenqingspring please update? :)

**Comment by freddymais** [comment]
> Could also change how it's imported:
> 
> ```diff
> - import * as animationData from './planets-spinning.json'
> + import animationData from './planets-spinning.json'
> ```
> 
> This way you don't have to get `animationData.default` as it automatically imports the default.

Sensacional!!! 
Uffaaaa.. ja tava desistindo.. 
Valeu demais!

**Comment by subodhk01** [comment]
@patrick-sung god

**Comment by jackdewhurst** [comment]
Update the docs!!!

**Comment by anearseeyou** [comment]
> should be
> `animationData: animationData.default`

nice

**Comment by vitpankin** [comment]
what about Typescript? it's not happy with that saying 
`Property 'default' does not exist on type '{ v: string; fr: number; ip: number; op: number; w: number; h: number; nm: string; ddd: number; assets: ({ id: string; `

**Comment by vitpankin** [comment]
Yeah, for TypeScript it's better to do 
`import { default as animationData } from '../animation-with-blackjack-and-hooks.json'`
and then

`  const defaultOptions:Options = {
    loop: true,
    autoplay: true, 
    animationData: animationData,
    rendererSettings: {
      preserveAspectRatio: 'xMidYMid slice'
    }
  };`

**Comment by wbdservices** [comment]
> should be `animationData: animationData.default`

where add this?

**Comment by raheesm** [comment]
> should be `animationData: animationData.default`

where should i add this
